### DISCUSSION TITLE
How to disable components animations?

### DISCUSSION THREAD

**Discussion by puntogris** [WalletConnect/walletconnect-monorepo#4062] [Developer Support]
Hello, is there a way to disable the animations? it makes the layout resize and looks weird.
These are the two components I'm using:
```
<w3m-network-button/>
<w3m-account-button/>
```

https://github.com/WalletConnect/walletconnect-monorepo/assets/45524597/38d67865-24a1-42c2-b1f7-68ff06e3d65d

**Comment by glitch-txs** [comment]
Hi, could you open an issue in [Web3Modal repository](https://github.com/WalletConnect/web3modal)? Thanks!
